src/components/players/SearchPlayers.js

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import classes from "./SearchPlayers.module.css";
+import PlayersSearchList from "./PlayersSearchList";
+import { getPlayersSearchResults } from "../../services/searchPlayerService";
+import LoadingDots from "../UI/LoadingDots";
+import { GoSearch } from "react-icons/go";
+
+const SearchPlayers = () => {
+  const [playersSearchList, setPlayersSearchList] = useState();
+  const [inputText, setInputText] = useState();
+  const [inputTextIsValid, setInputTextIsValid] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const getPlayersSearchMatches = async () => {
+      try {
+        if (inputTextIsValid) {
+          setIsLoading(true);
+          const data = await getPlayersSearchResults(inputText);
```

Comment:
I feel like getPlayersSearchResults could have been in the search file, it does not contain a log of logic.

also - for future search components that are retrieved from API calls - read about useDebounce :)